l10n/en/firefox/faq.ftl

```diff
@@ -0,0 +1,100 @@
+# This Source Code Form is subject to the terms of the { -brand-name-mozilla } Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/faq/
+
+# HTML page title
+page-title = There is a way to protect your privacy. Join { -brand-name-firefox }.
+page-desc = Whether you searched for a fast browser that protects your privacy, this FAQ is here to answer the most pressing { -brand-name-firefox }-related questions.
+
+firefox-faq = { -brand-name-firefox } FAQ
+whether-you-searched-privacy = Whether you searched for a fast browser that protects your privacy, this FAQ is here to answer the most pressing { -brand-name-firefox }-related questions.
+whether-you-searched-independant = Whether you searched for a fast browser, or you’re looking for independent tech that protects your privacy, this FAQ is here to answer the most pressing { -brand-name-firefox }-related questions.
+
+what-is-firefox = What is { -brand-name-firefox }?
+
+# Variables:
+# $url (url) - link to https://www.mozilla.org/firefox/browsers/
+# $url2 (url) - link to https://www.mozilla.org/firefox/products/
+
+the-firefox-browser = The { -brand-name-firefox-browser } is the only major browser backed by a non-profit that doesn’t sell your personal data to advertisers, while helping you protect your personal information. Learn more about the <a href="{ $url }">{ -brand-name-firefox-browsers }</a> and <a href="{ $url2 }">other products</a>.
+
+how-do-i = How do I get the { -brand-name-firefox-browser }?
+
+# Variables:
+# $url (url) - link to https://www.mozilla.org/firefox/new/
+# $url2 (url) - link to https://www.mozilla.org/firefox/windows/
+# $url3 (url) - link to https://www.mozilla.org/firefox/mac/
+# $url4 (url) - link to https://www.mozilla.org/firefox/linux/
+# $url5 (url) - link to https://www.mozilla.org/firefox/mobile/
+
+you-can-easily = You can easily <a href="{ $url }">download the { -brand-name-firefox } desktop browser</a> here. { -brand-name-firefox } works on <a href="{ $url2 }">{ -brand-name-windows }</a>, <a href="{ $url3 }">{ -brand-name-mac-short }</a> and <a href="{ $url4 }">{ -brand-name-linux}</a> devices, and is also available for <a href="{ $url5 }">{ -brand-name-android } and { -brand-name-ios }</a>. Make sure you’re downloading our browser from one of our trusted { -brand-name-mozilla }/{ -brand-name-firefox } pages.
```

Comment:
Copy (nit): The link should be "here".

<img width="412" alt="image" src="https://user-images.githubusercontent.com/2692333/96772336-9af34600-13a8-11eb-9388-b4be31916fe7.png">

<img width="640" alt="image" src="https://user-images.githubusercontent.com/2692333/96772341-9c247300-13a8-11eb-9c27-dca630385fc6.png">